tests/pages/firefox/accounts.py

```diff
@@ -22,4 +22,4 @@ def download_button(self):
 
     @property
     def create_account_button(self):
```

Comment:
You need to rename this to `is_create_account_form_displayed`, then it should work.

Reply:
D'oh, of course it was something obvious.